### PR TITLE
pypi & on-demand model download

### PR DESCRIPTION
**Pull request by movabo**
Add on-demand model download for a straight-forward usage from the pypi module.
Its published on https://pypi.org/project/NewsSentiment/

### CODE REVIEW

**Comment by movabo** [comment]
Tested the rebase from master by running:
```py
>>> from NewsSentiment import infer
>>> tsc = infer.TargetSentimentClassifier()
>>> tsc.infer(text_left="Whatever you think of ", target_mention="President Trump", text_right=", you have to admit that he’s an astute reader of politics.") 
```
No problems.

File: .idea/NewsMTSC.iml [hunk 1]
```diff
@@ -2,7 +2,10 @@
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="true">
     <exclude-output />
-    <content url="file://$MODULE_DIR$" />
+    <content url="file://$MODULE_DIR$">
+      <excludeFolder url="file://$MODULE_DIR$/DeepNewsSentiment.egg-info" />
```
Comment (fhamborg) [inline .idea/NewsMTSC.iml:6]: sollte es dann nicht NewsSentiment (statt Deep...) sein?

**Comment by movabo** [comment]
But now I wonder about the high deletion count which github displays (-59141).

**Review by fhamborg — CHANGES_REQUESTED** [review]
just one minor thing (unsure if its a problem, see my comment) - either way, pls merge after solving the "issue" (or not solving it)

**Comment by movabo** [comment]
Oh, it seems I deleted all datasets in one of the commits: https://github.com/fhamborg/NewsMTSC/commit/1459201b150de1482b2ae2fe82dc34b7ab19e406

**Comment by movabo** [comment]
Okay, now everything seems correct, no more deleted datasets and correct gitignores. .idea-settings updated to the current state of the project.
Ready to merge from my side. :)

**Pull request merged**